code/datums/supply_packs/supplies.dm

```diff
@@ -0,0 +1,502 @@
+	group = "Supplies"
+	containsdesc = "A formal carpet, able to cover about thirty square meters of area."
+
+/datum/supply_packs/arcade
+	name = "30 arcade tiles"
+	contains = list(/obj/item/stack/tile/arcade)
+	amount = 30
+	cost = 15
+	containertype = /obj/structure/closet/crate/basic
+	containername = "arcade tiles crate"
+	group = "Supplies"
+	containsdesc = "We're having fun tonight! Thirty pre-cut squares of fun, squishy, and slightly sticky arcade flooring."
+
+/datum/supply_packs/grass
+	name = "30 grass tiles"
+	contains = list(/obj/item/stack/tile/grass)
+	amount = 30
+	cost = 15
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "grass crate"
+	group = "Supplies"
+	containsdesc = "All-natural grass, sealed in a crate that preserves its freshness. Can cover about thirty square meters."
+
+/datum/supply_packs/watertank
+	name = "Water tank"
+	contains = list(/obj/structure/reagent_dispensers/watertank)
+	cost = 10
+	containertype = /obj/structure/largecrate
+	containername = "water tank crate"
+	group = "Supplies"
+	containsdesc = "A water tank, containing a little over six full buckets worth of water."
+
+/datum/supply_packs/fueltank
+	name = "Fuel tank"
+	contains = list(/obj/structure/reagent_dispensers/fueltank)
+	cost = 15
+	containertype = /obj/structure/largecrate
+	containername = "fuel tank crate"
+	group = "Supplies"
+	containsdesc = "A fuel tank, perfect for refueling welding tools. Warning, contents are volatile."
+
+/datum/supply_packs/silicatetank
+	name = "Silicate tank"
+	contains = list(/obj/structure/reagent_dispensers/silicate)
+	cost = 10
+	containertype = /obj/structure/largecrate
+	containername = "silicate tank crate"
+	group = "Supplies"
+	containsdesc = "A tank filled with silicate, perfect for repairing windows with."
+
+/datum/supply_packs/sacidtank
+	name = "Sulphuric acid tank"
+	contains = list(/obj/structure/reagent_dispensers/sacid)
+	cost = 15
+	containertype = /obj/structure/closet/crate/secure/large/reinforced/shard/empty
+	containername = "sulphuric acid tank crate"
+	group = "Supplies"
+	one_access = list(access_engine_minor, access_science)
+	containsdesc = "A tank filled to the brim with sulfuric acid. For when you're doing a lot of circuit board work, or you're a really thirsty alien."
+
+/datum/supply_packs/firefighting
+	name = "Firefighting equipment"
+	contains = list(/obj/item/clothing/suit/fire/firefighter,
+					/obj/item/clothing/suit/fire/firefighter,
+					/obj/item/clothing/mask/gas,
+					/obj/item/clothing/mask/gas,
+					/obj/item/clothing/head/hardhat/red,
+					/obj/item/clothing/head/hardhat/red,
+					/obj/item/weapon/extinguisher,
+					/obj/item/weapon/extinguisher,)
+	cost = 20
+	containertype = /obj/structure/closet/crate/basic
+	containername = "firefighting equipment crate"
+	group = "Supplies"
+	containsdesc = "Firefighting gear. Includes two full suits to protect upcoming firemen against the dangers of fire. Fire axe not included."
```

Comment:
Oh never mind, this is where the regular non-advanced firefighting equipment was.